### ISSUE TITLE
Bad performance on CUHK03-NP 

### ISSUE THREAD

**Issue by LQ199302** [layumi/Person_reID_baseline_pytorch#43]
I tried this code on CUHK03-NP dataset, but it didn't work well, and only arrived rank 1: 61.6 , mAp: 56.7.

**Comment by layumi** [comment]
Hi @LQ199302, 
I think it is a relative high result. You may check the state-of-the-art performance at https://github.com/zhunzhong07/person-re-ranking/tree/master/CUHK03-NP#state-of-the-art.
CUHK03-NP is more difficult than DukeMTMC-reID and Market-1501

**Comment by LQ199302** [comment]
OK, thanks a lot

**Comment by EricWu123** [comment]
I tried this code on CUHK03-NP dataset, but I got a lower result,just rank1:46.6 mAP:42.6. I set batch size 32, and learning rate keep unchanged (as same as the original code). I want to know if you change other hyperparameters.

**Comment by LQ199302** [comment]
OK, in fact, I don't only use PCB, I try to consider the influence of global feature and combine both of them.

**Comment by EricWu123** [comment]
OK, thanks a lot.

**Comment by SpONA** [comment]
> I tried this code on CUHK03-NP dataset, but it didn't work well, and only arrived rank 1: 61.6 , mAp: 56.7.

could you please show the training settings of cuhk03-np? Thanks~
